.travis.yml

```diff
@@ -28,36 +10,24 @@ before_install:
 - export PATH=${PATH}:./vendor/bundle
 
 install:
-- rvm use 2.3.8 --install --fuzzy
-- gem update --system
-- gem install sass
-- gem install ruby_dep -v 1.3.1
-- gem install jekyll -v 3.4.3
-- gem install jekyll-redirect-from
-- nvm install 6
-- nvm use 6
-- node --version
-
-before_script:
-  - sudo chmod +x /usr/local/bin/sbt
+  # Disabled due to unrelated conflicts running the build in travis
```

Comment:
See https://travis-ci.org/47deg/github4s/jobs/626332884#L525-L530 to see the problem